ca-service/src/test/java/uk/sky/carapplication/controller/PrivateControllerTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.sky.carapplication.controller;
+
+import org.junit.jupiter.api.*;
+import org.mockito.Mock;
+import uk.sky.carapplication.service.HeadersService;
+
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class PrivateControllerTest {
+
+    private PrivateController privateController;
+    @Mock
+    HeadersService headersService;
+
+    @BeforeEach
+    public void setUp(){
+        privateController = new PrivateController(headersService);
+    }
```

Comment:
Have a look into @Injectmocks annotation. Its quite a handy little annotation and will save you from writing lines of code to initialise the object and pass in a mock